src/microbe_stage/systems/MicrobeDeathSystem.cs

```diff
@@ -140,13 +140,18 @@ public sealed class MicrobeDeathSystem : AEntitySetSystem<float>
             var positionAdded = new Vector3(random.Next(-2.0f, 2.0f), 0,
                 random.Next(-2.0f, 2.0f));
 
+            var chunkScale = 1f;
+
+            if (isBacteria)
+                chunkScale = 0.5f;
+
             var chunkType = new ChunkConfiguration
             {
-                ChunkScale = 1.0f,
+                ChunkScale = chunkScale,
                 Dissolves = true,
                 PhysicsDensity = 1200.0f,
-                Radius = 1.0f,
-                Size = 3.0f,
+                Radius = chunkScale,
+                Size = chunkScale,
```

Comment:
Do we want to factor in the hex size of the chunk to the engulfment size?

Reply:
I wanted to make it look so if chunk is smaller than microbe visuals, it is engulfable

Reply:
Shouldn't this still be `3 * chunkScale` otherwise this will massively buff chunk eating as you can fit 3x more chunks in your cell than before? (this line exactly Radius or ChunkScale lines shouldn't be changed).